packages/vite/bins/rw-vite-dev.mjs

```diff
@@ -2,9 +2,9 @@
 import { createServer } from 'vite'
 import yargsParser from 'yargs-parser'
 
-import projectConfig from '@redwoodjs/project-config'
```

Comment:
This is an mjs file so it would import the ESM version of `@redwoodjs/project-config`. The ESM version has no default export. I could've re-written this as 

```
import * as projectConfig from '@redwoodjs/project-config'
```

but didn't see the point when we don't do that anywhere else